src/shared/post-processors/exchange-rates/price-routing-base.ts

```diff
@@ -0,0 +1,6 @@
+import { Context } from '@processor'
+import { Currency } from '@shared/post-processors/exchange-rates/currencies'
+
+export const getBasePrice = async (ctx: Context, height: number, base: Currency, quote: Currency) => {
+  return undefined
```

Comment:
WIP probably

Reply:
yes, not yet sure we even need any rates like this there yet

Reply:
how to say something simple with many words